tests/tests/test-contract-precompiled.ts

```diff
@@ -0,0 +1,44 @@
+import { expect } from "chai";
+
+import { createAndFinalizeBlock, customRequest, describeWithMoonbeam } from "./util";
+import { AbiItem } from "web3-utils";
+
+describeWithMoonbeam("Moonbeam RPC (Contract of precompiled function)", `simple-specs.json`, (context) => {
+	const GENESIS_ACCOUNT = "0x6be02d1d3665660d22ff9624b7be0551ee1ac91b";
+	const GENESIS_ACCOUNT_PRIVATE_KEY = "0x99B3C12287537E38C90A9219D4CB074A89A16E9CDB20BF85728EBD97C343E342";
+
+    const TEST_CONTRACT_ABI = {
+        constant: false,
+        // keep in mind that `testit(bytes8)` is the signature given here,
+        // this will affect the data that will be transmitted to the function
+        // in this case, assuming the data is 0xdeadbeef, we calculate (using pyethereum):
+        // >>> ethereum.utils.sha3("testit(bytes8)").hex()
+        // which will result in 0xba9cf00f603084017b95e68b576cac531a2fcba78b2b0773614428d7905c09ff
+        // and the first 4 bytes of that (the function signature) will be part of that
```

Comment:
I don't understand what you can alter in the precompiles that you can't on a normal contract, can you explain again ?